src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -6759,6 +6759,47 @@ def test_aks_create_with_azurecontainerstorage(self, resource_group, resource_gr
             self.is_empty(),
         ])
 
+    # live only due to downloading k8s-extension extension
+    @live_only()
```

Comment:
Why does the installation of k8s-extension extension need to mark this test as `@live_only()`?

Reply:
This was suggested by @FumingZhang in my previous PR here
https://github.com/Azure/azure-cli-extensions/pull/6837#discussion_r1360196353

Reply:
Got it, thanks~